libevm/interop/invoke.go

```diff
@@ -0,0 +1,64 @@
+package interop
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/ethereum/go-ethereum/log"
+	"reflect"
+)
+
+func Invoke(target interface{}, method string, args string) string {
+	log.Debug(">> invoke", "method", method, "args", args)
+	result := toJsonResponse(callMethod(target, method, args))
+	log.Debug("<< response", "result", result)
```

Comment:
Do we still want to log every single invocation? Probably has performance implications as the log level is not known to libevm.